src/main/kotlin/no/nav/familie/ks/sak/integrasjon/sanity/SanityQueries.kt

```diff
@@ -0,0 +1,36 @@
+package no.nav.familie.ks.sak.integrasjon.sanity
+
+// TODO: Tilpass query for KS
+const val hentBegrunnelser =
+    "*[_type == \"begrunnelse\" && behandlingstema != \"EØS\" && apiNavn != null && navnISystem != null]{" +
+        "apiNavn," +
+        "navnISystem," +
+        "hjemler," +
+        "hjemlerFolketrygdloven," +
+        "vilkaar," +
+        "rolle," +
+        "lovligOppholdTriggere," +
+        "bosattIRiketTriggere," +
+        "giftPartnerskapTriggere," +
+        "borMedSokerTriggere," +
+        "ovrigeTriggere," +
+        "endretUtbetalingsperiodeTriggere," +
+        "endretUtbetalingsperiodeDeltBostedUtbetalingTrigger," +
+        "endringsaarsaker," +
+        "utvidetBarnetrygdTriggere" +
```

Comment:
Trenger ikke "utvidetBarnetrygdTriggere", tror jeg. Mangler trigger for Barnehageplass

Reply:
Ja, denne må vi se mer på når vi får definert ny begrunnelse-mal i Sanity 👍